webapp/Makefile

```diff
@@ -44,12 +45,7 @@ install-domserver:
 			ln -s $$realtarget $$link ; \
 		fi \
 	done
-	-for d in cache logs sessions ; do \
-		$(INSTALL_WEBSITE) -m 0775 -d $(DESTDIR)$(domserver_webappdir)/var/$$d ; \
-	done
-	-$(INSTALL_WEBSITE) -t $(DESTDIR)$(domserver_webappdir)/var var/*.php var/*.cache
-	cp        -t $(DESTDIR)$(domserver_webappdir) phpunit.xml.dist
-	chmod a+r    $(DESTDIR)$(domserver_webappdir)/phpunit.xml.dist
+	$(INSTALL_DATA) -t $(DESTDIR)$(domserver_webappdir) phpunit.xml.dist
```

Comment:
Do we need to install phpunit.xml at all? Is it not only for dev?

Reply:
It's used in Gitlab CI, which does a plain install (not maintainer-mode), so we can't simply delete it, or only install in maintainer-mode. Any alternatives to leaving this as is?

Reply:
It does not seem worth it to jump many hoops for not installing this small file, does not really hurt to keep it